### ISSUE TITLE
Make a components library

### ISSUE THREAD

**Issue by Andrewnt219** [Andrewnt219/rent-near-me#75]
Let's pick a component library and gradually make our own version with Tailwindcss.

1. Infima https://infima.dev/docs/components/card

2. Flowbite
https://flowbite.com/docs/getting-started/introduction/

3. Vercel UI
https://react.geist-ui.dev/en-us

**Comment by vitokhangnguyen** [comment]
Just curious, why Material UI is not on the list? 🤔 Not that I prefer it, just wanna know why

**Comment by Andrewnt219** [comment]
Material-UI effects like ripple are hard to implement, and their HTML/CSS is a pain to reverse engineering.

**Comment by vitokhangnguyen** [comment]
> Material-UI effects like ripple are hard to implement, and their HTML/CSS is a pain to reverse engineering.

True, but they give a very robust way to customize their component, no? Like, we don't have to reverse engineer it if we use it. Again, I don't like it either, just wondering...

**Comment by Andrewnt219** [comment]
True, but a robust configuration is for open-source libraries (i.e. used in a very wide range of different projects). It'll be faster for development and easier to use if the components are more opinionated.

**Comment by vitokhangnguyen** [comment]
> True, but a robust configuration is for open-source libraries (i.e. used in a very wide range of different projects). It'll be faster for development and easier to use if the components are more opinionated.

Well, my company uses it and they uses it pretty well so that's why I'm asking 😂

**Comment by Andrewnt219** [comment]
We can have material-ui system but styling is from a different library.

**Comment by vitokhangnguyen** [comment]
> We can have material-ui system but styling is from a different library.

Nah, let's not do Mat UI at all

**Comment by Andrewnt219** [comment]
Ok, I'm going with [Radix UI](https://www.radix-ui.com/) for accessibility and styling from [Flowbite](https://flowbite.com/docs/getting-started/introduction/)
